tutorials/models/1_gnn/9_gat.py

```diff
@@ -1,40 +1,37 @@
 """
 .. _model-gat:
 
-Understand Graph Attention Network
+Graph attention network
 ==================================
 
 **Authors:** `Hao Zhang <https://github.com/sufeidechabei/>`_, `Mufei Li
 <https://github.com/mufeili>`_, `Minjie Wang
 <https://jermainewang.github.io/>`_  `Zheng Zhang
 <https://shanghai.nyu.edu/academics/faculty/directory/zheng-zhang>`_
 
-From `Graph Convolutional Network (GCN) <https://arxiv.org/abs/1609.02907>`_,
-we learned that combining local graph structure and node-level features yields
-good performance on node classification task. However, the way GCN aggregates
-is structure-dependent, which may hurt its generalizability.
+In this tutorial, you learn about a graph attention network (GAT) and how it can be 
+implemented in MXNet-DGL. You can also learn to visualize and understand what the attention 
```

Comment:
The code is implemented in PyTorch rather than MXNet.

Reply:
Thanks for the review. Updated the commit.